index.md

```diff
@@ -0,0 +1,29 @@
+## Malagasy artists:
+Welcome to this list of malagasy artists. If you want to know the famous artists in madagascar, just click those link below.
```

Comment:
Proper nouns like `malagasy` and `madagascar` need to have capital letters at the start.